patterns/link-list.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Title: Link list
+ * Slug: ucsc-2022/link-list
+ * Block Types: core/list
+ * Categories: text, columns
+ */
+?>
+<!-- wp:group {"backgroundColor":"white","textColor":"black","className":"ucsc__link-list","layout":{"type":"constrained"}} -->
+<div class="wp-block-group ucsc__link-list has-black-color has-white-background-color has-text-color has-background"><!-- wp:heading {"lock":{"move":true,"remove":true},"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"textColor":"black","className":"ucsc__link-list-header","fontSize":"two"} -->
```

Comment:
Let's move this to the group wrapper `ucsc__link-list` so all elements can be controlled if need be.

Reply:
That class is on the outer-most HTML element of the block, @Vinsanity . I'm not sure what you mean.